From d342961f680e6e8b5f7ad8f1d43fadf15872da5d Mon Sep 17 00:00:00 2001
From: Assaf Paz <damagedspline@gmail.com>
Date: Fri, 7 May 2021 22:24:07 +0300
Subject: [PATCH] XMBC MVC compatibility

---
 CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d0ea7da..bd9f2e3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,6 +40,7 @@ if (CMAKE_SYSTEM_NAME MATCHES "Windows")
     src/mfx_load_plugin.cpp
     src/mfx_plugin_hive.cpp
     src/mfx_win_reg_key.cpp
+    src/mfx_driver_store_loader.cpp
   )
 endif (CMAKE_SYSTEM_NAME MATCHES "Windows")
 
@@ -55,7 +56,7 @@ add_definitions(
 
 configure_file (${CMAKE_SOURCE_DIR}/libmfx.pc.cmake ${CMAKE_BINARY_DIR}/libmfx.pc @ONLY)
 
-add_library( mfx STATIC ${SOURCES} )
+add_library( libmfx STATIC ${SOURCES} )
 install (DIRECTORY ${CMAKE_SOURCE_DIR}/mfx DESTINATION ${CMAKE_INSTALL_PREFIX}/include FILES_MATCHING PATTERN "*.h")
 install (FILES ${CMAKE_BINARY_DIR}/libmfx.pc DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/pkgconfig)
-install (TARGETS mfx ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
+install (TARGETS libmfx ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib)
-- 
2.30.0.windows.2

